app/(dashboard)/[storeId]/(routes)/settings/page.tsx

```diff
@@ -0,0 +1,39 @@
+import prismadb from "@/lib/db";
+import { auth } from "@clerk/nextjs/server";
+import { redirect } from "next/navigation";
```

Comment:
```suggestion
```suggestion
const params = useParams(); // use params hook
    const router = useRouter(); // use router hook
    const origin = useOrigin(); // use origin hook
```
```